src/provider/index.ts

```diff
@@ -1,5 +1,7 @@
 import * as slack from './slack';
+import * as oauth from './oauth';
 
 export * as slack from './slack';
+export * as oauth from './oauth';
 
 export type INative = slack.INative;
```

Comment:
Include an oauth.INative here that publishes as any if necessary.

Reply:
I am not sure what you mean by "include". This module exports a single INative type that is hardwired to that of Slack.

Reply:
export type INative = slack.INative | oauth.INative;

Reply:
ok added `| any`